.travis-mirage.sh

```diff
@@ -0,0 +1,55 @@
+## mirage setup
+
+set -ex
+
+## fetch+execute the OCaml/opam setup script
+wget https://raw.githubusercontent.com/mor1/ocaml-travisci-skeleton/master/.travis-ocaml.sh
```

Comment:
need to rename the use to `ocaml` instead of `mor1`. I'll merge the path after that!